src/axom/slic/core/Logger.cpp

```diff
@@ -58,6 +62,18 @@ Logger::~Logger()
   }  // END for all levels
 }
 
+//------------------------------------------------------------------------------
+void Logger::setAbortFunction(void (*abort_func)(void))
+{
+  if(abort_func == nullptr)
+  {
+    std::cerr << "WARNING: supplied abort function is NULL!\n";
```

Comment:
Suggestion: Please clarify this error message to let the user know where the problem is, 
e.g. something like:
> WARNING: slic::Logger::setAbortFunction() -- Supplied abort function was invalid